### PR TITLE
chore: update kubernetes to python3

### LINKED ISSUE #4734: Kubernetes still uses python2
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-server

**Current behavior:**

Kurbernetes run script uses python2 instead of python3
**Expected behavior:**

It should execute python3 instead of python

**Related code:**

https://github.com/fossasia/open-event-server/blob/224e679c65db9bbc877e84678ba38adfd07caab9/kubernetes/run.sh#L10

### PR DESCRIPTION
**Pull request by srv-twry**
Fixes #4734 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Kubernetes was still using python2.

#### Changes proposed in this pull request:
Since we have updated to python3, Updated the kubernetes deployment settings to use python3

### CODE REVIEW

**Comment by poush** [comment]
Are you able to successfully deploy on kubernetes using this?

**Comment by srv-twry** [comment]
@poush No, I didn't try it.
The problem is that no one has been able to deploy it. Here no one means all the gsoc students. 
I'll try again and ask for help.

File: kubernetes/images/web/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:2-slim
+FROM python:3.5-slim
```
Comment (iamareebjamal) [inline kubernetes/images/web/Dockerfile]: This should be latest, i.e., 3.6-slim

Comment (schedutron) [reply kubernetes/images/web/Dockerfile]: 👍

**Comment by srv-twry** [comment]
@niranjan94 @poush @iamareebjamal I am able to deploy it on kubernetes but unable to access it due to a server error. 
Here is the full stacktrace:
```
ProgrammingError: (psycopg2.ProgrammingError) relation "mails" does not exist
LINE 2: FROM mails 
             ^
 [SQL: 'SELECT mails.id AS mails_id, mails.recipient AS mails_recipient, mails.time AS mails_time, mails.action AS mails_action, mails.subject AS mails_subject, mails.message AS mails_message \nFROM mails \n LIMIT %(param_1)s'] [parameters: {'param_1': 1}]
  File "app/views/healthcheck.py", line 70, in check_migrations
    db.session.query(model).first()
  File "sqlalchemy/orm/query.py", line 2755, in first
    ret = list(self[0:1])
  File "sqlalchemy/orm/query.py", line 2547, in __getitem__
    return list(res)
  File "sqlalchemy/orm/query.py", line 2855, in __iter__
    return self._execute_and_instances(context)
  File "sqlalchemy/orm/query.py", line 2878, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "sqlalchemy/engine/base.py", line 945, in execute
    return meth(self, multiparams, params)
  File "sqlalchemy/sql/elements.py", line 263, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "sqlalchemy/engine/base.py", line 1053, in _execute_clauseelement
    compiled_sql, distilled_params
  File "sqlalchemy/engine/base.py", line 1189, in _execute_context
    context)
  File "sqlalchemy/engine/base.py", line 1402, in _handle_dbapi_exception
    exc_info
  File "sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "sqlalchemy/engine/base.py", line 1182, in _execute_context
    context)
  File "sqlalchemy/engine/default.py", line 470, in do_execute
    cursor.execute(statement, parameters)
```

It says that the relation "mails" does not exist. Mail is well defined according to me. I tried researching for it but couldn't find anything. Can you please take a look ?

Sentry link: https://sentry.io/share/issue/bfe4b7b6314e4081ae303ccebb9b069f/

**PS**
I was facing this issue before this PR i.e. it doesn't have anything to do with updating to python3.

**Comment by srv-twry** [comment]
Closing this as it is not on agenda right now and it is unnecessarily adding to the PR queue. Will reopen when I am able to figure out the way to fix the problem

**Pull request closed without merging**